doc/changelog.rst

```diff
@@ -1,5 +1,6 @@
 Changelog
 =========
+* :feature:`925` Support light-mode by changing ``log.console_theme.dark_background`` configuration
```

Comment:
`Support was added for terminals with light backgrounds...`

Reply:
@vmalloc Fixed, please review again